content/blog/pulumi-components/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "The New Generation of Pulumi Components"
```

Comment:
Should we emphasize the use case? E.g.

```suggestion
title: "Announcing the Next-Generation of Pulumi Components: Enabling Infrastructure Abstractions"
```

A bit long ... but I'd love to focus a bit more on the "why" (use case) rather than just the "what" of the feature.